src/mscorlib/corefx/System/Globalization/CalendarData.Unix.cs

```diff
@@ -38,10 +67,43 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
 
         private static bool SystemSupportsTaiwaneseCalendar()
         {
-            // TODO: Implement this.
-            return false;
+            return true;
         }
 
         // PAL Layer ends here
+
+        private static bool GetCalendarInfo(string localeName, CalendarId calendarId, CalenderDataType dataType, out string calendarString)
+        {
+            const int stringSize = 80;
```

Comment:
It was copied from the CalendarData.Win32.cs file:
```C#
        private static unsafe bool CallGetCalendarInfoEx(string localeName, CalendarId calendar, uint calType, out string data)
        {
            const int BUFFER_LENGTH = 80;

            // The maximum size for values returned from GetCalendarInfoEx is 80 characters.
            char* buffer = stackalloc char[BUFFER_LENGTH];
```

Since this is the max Windows is going to return, I felt it was a safe max length for Linux.